src/status_im/contexts/wallet/common/collectibles_tab/view.cljs

```diff
@@ -28,9 +28,10 @@
                                (on-long-press collectible))}]))
 
 (defn view
-  [{:keys [theme collectibles filtered? on-collectible-press on-end-reached current-account-address
+  [{:keys [collectibles filtered? on-collectible-press on-end-reached current-account-address
            on-collectible-long-press]}]
-  (let [no-results-match-query? (and filtered? (empty? collectibles))]
+  (let [no-results-match-query? (and filtered? (empty? collectibles))
+        theme                   (quo.theme/use-theme)]
```

Comment:
Why does removing theme prop fixes the issue? 
What's the root problem here?

Reply:
We removed the `theme` prop in the entire app, and in the newer update, the `theme` should be imported from the hook. Someone had migrated the entire `theme` from the prop to the hook but has forgot here. That's the reason of the issue.

Reply:
@mmilad75, but how this change affects the appearance of the spinner?

Reply:
The issue was that `theme` was null before, so the image component was receiving a wrong image source, so it didn't have anything to load, therefore, it was showing the spinner instead of the actual image forever. Now, the value of `theme` is either `:dark` or `:light`. This will give the image component a true source value.